DigitalLearningSolutions.Data.Tests/DataServices/UserDataServiceTests.cs

```diff
@@ -367,5 +367,40 @@ public void GetNumberOfActiveApprovedDelegatesAtCentre_returns_expected_count()
             // Then
             count.Should().Be(3420);
         }
+
+        [Test]
+        public void GetDelegateUserCardsByCentreId_populates_DelegateUser_fields_correctly()
+        {
+            // Given
+            var expected = UserTestHelper.GetDefaultDelegateUser(
+                dateRegistered: DateTime.Parse("2010-09-22 06:52:09.080"),
+                jobGroupName: "Nursing / midwifery"
+            );
+
+            // When
+            var userCards = userDataService.GetDelegateUserCardsByCentreId(2);
+
+            // Then
+            var userCard = userCards.Single(user => user.Id == 2);
+            userCard.Should().BeEquivalentTo(expected);
+            userCard.Active.Should().BeTrue();
+            userCard.SelfReg.Should().BeFalse();
+            userCard.ExternalReg.Should().BeFalse();
+            userCard.AdminId.Should().BeNull();
+        }
+
+        [Test]
+        public void GetDelegateUserCardsByCentreId_populates_DelegateUserCard_fields_correctly()

```

Comment:
These tests have pretty much the same names - could we add "withLinkedAdminAccount" or something similar to name here?

Reply:
Change `_fields_` to `_admin_fields_` to distinguish and avoid name becoming too long